app/bot/openai/openai.go

```diff
@@ -180,18 +182,48 @@ func (o *OpenAI) checkResponseAI(username, responseAI string) (ok bool, banMessa
 
 // Help returns help message
 func (o *OpenAI) Help() string {
-	return genHelpMsg(o.ReactOn(), "Спросите что-нибудь у ChatGPT")
+	return bot.GenHelpMsg(o.ReactOn(), "Спросите что-нибудь у ChatGPT")
 }
 
 func (o *OpenAI) chatGPTRequest(request, userPrompt, sysPrompt string) (response string, err error) {
+	// The API supports 4097 tokens ~16000 characters (<=4 per token) for request + result together
+	// The response is limited to 1000 tokens and OpenAI always reserved it for the result
+	// So the max length of the request should be 3000 tokens or ~12000 characters
+	defaultReducer := func(text string) (result string) {
```

Comment:
i think moving the body of code directly to reduceRequest will make it more readable.